CHANGELOG.md

```diff
@@ -23,7 +19,9 @@
 
 **Bug Fixes**:
 
+- Fix performance regression in disk spooling by using page counts to estimate the spool size. ([#3379](https://github.com/getsentry/relay/pull/3379))
 - Allow spans with no `exclusive_time`. ([#3385](https://github.com/getsentry/relay/pull/3385))
+- Perform clock drift normalization only when `sent_at` is set in the `Envelope` headers. ([#3405](https://github.com/getsentry/relay/pull/3405))
```

Comment:
Please move Bug Fixes to above Features, it's the first thing people should read IMO.

Reply:
Yep, will do.